src/main/resources/add-ons-to-index.json

```diff
@@ -4024,6 +4138,9 @@
       "uid": "org.openmrs.module.omod-export-module",
       "type": "OMOD",
       "name": "Omod Export Module",
+      "tags": [
+        "export"
```

Comment:
Get rid of "export". Add "sysadmin" (or something else similar if there's already a label like that)